packages/adena-extension/src/components/transfer/address-input/address-input.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import AddressInput, { type AddressInputProps } from './address-input';
+import { Meta, StoryObj } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'components/transfer/AddressInput',
+  component: AddressInput,
+} as Meta<typeof AddressInput>;
+
+const addressBookInfos = [
```

Comment:
Is this actually used in the storybook?

Reply:
This variable is the address book information when the 'opened' option is enabled.